stone/target/python_rsrc/stone_base.py

```diff
@@ -34,6 +34,16 @@ def __init__(self, tag, value=None):
         self._tag = tag
         self._value = value
 
+    def __eq__(self, other):
+        # Also need to check if one class is a subclass of another. If one union extends another,
+        # the common fields should be able to be compared to each other.
+        return (
+            (isinstance(self, other.__class__) or isinstance(other, self.__class__)) and
```

Comment:
Oh, that makes sense.  Though now I'm a bit worried about "identical" values having different underlying classes.

When someone instantiates says `B.t0`, maybe we should return `A.t0`?